### PR TITLE
Adding multi choice Votes

### PR DESCRIPTION
**Pull request by navro**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -28,3 +28,16 @@
 //= require pickers
 //= require cookies_eu
 //= require cocoon
```
Comment (davidwessman) [inline app/assets/javascripts/application.js]: I think this should be put in it's own javascript file. (I like how this critique applies so much to me as well... ;) )

File: app/services/vote_service.rb [hunk 2]
```diff
@@ -1,10 +1,13 @@
 module VoteService
-  def self.user_vote(post, option)
+  def self.user_vote(post)
     begin
       VotePost.transaction do
         post.save!
-        option.count += 1
-        option.save!
+        options = VoteOption.find(post.vote_option_ids)
+        options.each do |o|
```
Comment (davidwessman) [inline app/services/vote_service.rb]: How is the amount of votes limited to the `choices`?

**Comment by davidwessman** [comment]
Looking really good! Just some questions

File: app/services/vote_service.rb (hunk 2 above)
Comment (navro) [inline app/services/vote_service.rb]: > How is the amount of votes limited to the choices?

We set `choices` when we create/edit a `Vote`. When we run `post.save!` the validation `:option_details` in `VotePost` checks that `vote_option_ids.count <= vote.choices` and that no duplicates are included :)

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (navro) [inline app/assets/javascripts/application.js]: > I think this should be put in it's own javascript file. (I like how this critique applies so much to me as well... ;) )

:+1:

File: app/models/vote_post.rb [hunk 3]
```diff
@@ -21,4 +21,17 @@ def vote_open
       errors.add(:votecode, I18n.t('vote_post.vote_closed'))
     end
   end
+
+  def option_details
+    if vote_option_ids.present?
+      unless vote_option_ids.count <= vote.choices
+        errors.add(:vote_option_ids, I18n.t('vote_post.too_many_options'))
```
Comment (davidwessman) [inline app/models/vote_post.rb]: Not completely sure if these should return false when failing.

File: app/services/vote_service.rb (hunk 2 above)
Comment (davidwessman) [inline app/services/vote_service.rb]: :+1:

**Comment by davidwessman** [comment]
Looking really good! :+1:

**Pull request merged**